src/_pytest/config/findpaths.py

```diff
@@ -87,6 +87,7 @@ def make_scalar(v: object) -> Union[str, List[str]]:
 
 
 def locate_config(
+    invocation_dir: Path,
```

Comment:
```suggestion
    fallback_search_path: Path,
```
i'm not happy with my first suggestion tho,

Reply:
closing as the prior discussion applies